default.nix

```diff
@@ -184,6 +184,16 @@ let
     };
 
     plutus-playground = rec {
+      documentation-site = let
+        # TODO: the playgroundUrl needs to be set to whatever will actually be appropriate when it's bundled
+        # with the playground
+        adjustedTutorial = docs.plutus-tutorial.override { playgroundUrl = "../.."; haddockUrl = "../haddock"; };
```

Comment:
since this is being deployed to the same domain, can't you just use relative paths?

Reply:
Those are relative paths, no? I just mean we'll need to tweak it for whatever the exact layout.

Reply:
I mean setting the `playgroundUrl`, why would we do that? Can you give me an example.

Reply:
The `playgroundUrl` is for the docs to link to the playground. But we could put the playground in a number of places, e.g. we could put it at the root or at `/playground`, say.

It's natural to make it configurable from the documentation's point of view, so we just have to say what it actually is here.

Reply:
got it